automation/README.md

```diff
@@ -12,6 +12,7 @@
 - Use prefix action functions according to element type.
   - `click` for links and buttons.
   - `select` for elements with options like dropdowns, radios, and checkboxes.
+- Use test flows for repeated logic and assertions. Refer to example `test_flows.md`
```

Comment:
If you look at the other examples you can actually create a direct link here, look at the bottom of this file for examples on how to create it